Update to 1.21.5

granny opened the pull request:
Currently compiles. Haven't tested to see if it actually works yet. Some features may or may not throw errors, run at your own risk.

PR Blockers: 

- [x] https://github.com/KyoriPowered/adventure-platform/pull/206
- [x] https://github.com/KyoriPowered/adventure-platform-mod/pull/197
- [x] register new vanilla blocks
<!-- bot: pl3xmap-pr-build -->
---
Download the pl3xmap jar for this pull request: [pl3xmap-89.zip](https://nightly.link/granny/Pl3xMap/actions/artifacts/3145661835.zip)

granny committed:
it compiles ¯\_(ツ)_/¯

granny committed:
Merge branch 'v3' into chore/1.21.5

granny committed:
stop using early version of paper dev bundle

granny committed:
remove TODO

tconnard commented:
Completely borked until adventure library is 1.21.5 compatible.

granny commented:
> Completely borked until adventure library is 1.21.5 compatible, at least on fabric.

Looks like they have PRs for 1.21.5 that are being worked on. I've listed them in the PR description.

granny committed:
check for world keys that have the colon replaced with a dash in sse endpoint, closes #91

Kesuaheli commented:
The pull request for the 1.21.5 update of the adventure mod is now merged (listed in pr blockers). But they haven't released 6.4.0 yet.

Is it still a blocker?

BillyGalbreath commented:
> The pull request for the 1.21.5 update of the adventure mod is now merged (listed in pr blockers). But they haven't released 6.4.0 yet.
> 
> Is it still a blocker?

notice there are two checkboxes in the list of pr blockers ;)

granny committed:
bump gradle to 8.13

granny committed:
temoprarily exclude ansi from adventure-plaform-fabric

tinyremapper keeps trying to remap ansi's SystemConsole class that's compiled for java 22. will need to look into this deeper at a later date. Might just be an issue with how the build scripts are formatted

granny committed:
bump fabric, adventure and adventure fabric versions

granny committed:
Merge branch 'v3' into chore/1.21.5

granny committed:
bump adventure-bukkit

granny committed:
update version in README

granny committed:
register missing vanilla blocks

granny committed:
respect new dry_foliage_color biome field

granny committed:
update lang resource files

granny committed:
update build reference in README

granny committed:
Merge branch 'v3' into chore/1.21.5

granny committed:
update build reference in README
